django/cantusdb_project/main_app/templates/content_overview.html

```diff
@@ -0,0 +1,80 @@
+    <table class="table table-sm small table-bordered">
+        <small>Displaying the <b>50</b> most recently updated objects</small>
+        <div style="height:10px;"></div>
+        <thead>
+            <tr>
+                <th scope="col" class="text-wrap" style="text-align:center">Title / Manuscript Full Text / Name</th>
+                <th scope="col" class="text-wrap" style="text-align:center">Type</th>
+                <th scope="col" class="text-wrap" style="text-align:center">Creation Date</th>
+                <th scope="col" class="text-wrap" style="text-align:center">Creator</th>
+                <th scope="col" class="text-wrap" style="text-align:center">Last Updated Date</th>
+                <th scope="col" class="text-wrap" style="text-align:center">Last Updated By</th>
+                <th scope="col" class="text-wrap" style="text-align:center">Operations</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for object in objects %}
+                <tr>
+                    {% if object.title %}
+                        <td class="text-wrap" style="text-align:center">
+                            <a href="{{ object.get_absolute_url }}"><b>{{ object.title|truncatechars:30 }}</b></a>
+                        </td>
+                    {% elif object.manuscript_full_text_std_spelling %}
+                        <td class="text-wrap" style="text-align:center">
+                            <a href="{{ object.get_absolute_url }}"><b>{{ object.manuscript_full_text_std_spelling|truncatechars:30 }}</b></a>
+                        </td>
+                    {% elif object.name %}
+                        <td class="text-wrap" style="text-align:center">
+                            {% if object|classname == "Notation" or object|classname == "Provenance" or object|classname == "Segment" %}
+                                <b>{{ object.name|truncatechars:30 }}
+                            {% else %}
+                                <a href="{{ object.get_absolute_url }}"><b>{{ object.name|truncatechars:30 }}</b></a>
+                            {% endif %}
+                        </td>
+                    {% elif object.first_name %}
+                        <td class="text-wrap" style="text-align:center">
+                            <a href="{{ object.get_absolute_url }}"><b>{{ object.first_name }} {{ object.family_name }}</b></a>
+                        </td>
+                    {% endif %}
+                    <td class="text-wrap" style="text-align:center">{{ object|classname }}</td>
+                    <td class="text-wrap" style="text-align:center">{{ object.date_created|date:'Y-m-d H:i' }}</td>
+                    <td class="text-wrap" style="text-align:center">
+                        {% if object.created_by is None %}
+                            {{ object.created_by }}
+                        {% else %}
+                            <a href="/admin/users/user/{{ object.created_by.id }}/change/">
```

Comment:
Did you look into ways to avoid hardcoding the urls? Can the `{% url ... %}` tag be used to generate admin urls as well?